pytensor/link/pytorch/dispatch/slinalg.py

```diff
@@ -0,0 +1,95 @@
+import torch
+
+from pytensor.link.pytorch.dispatch.basic import pytorch_funcify
+from pytensor.tensor.slinalg import (
+    BlockDiagonal,
+    Cholesky,
+    Eigvalsh,
+    Solve,
+    SolveTriangular,
+)
+
+
+@pytorch_funcify.register(Eigvalsh)
+def pytorch_funcify_Eigvalsh(op, **kwargs):
+    if op.lower:
+        UPLO = "L"
+    else:
+        UPLO = "U"
+
+    def eigvalsh(a, b):
+        if b is not None:
+            raise NotImplementedError(
+                "torch.linalg.eigvalsh does not support generalized eigenvector problems (b != None)"
+            )
+        return torch.linalg.eigvalsh(a, UPLO=UPLO)
+
+    return eigvalsh
+
+
+@pytorch_funcify.register(Cholesky)
+def pytorch_funcify_Cholesky(op, **kwargs):
+    upper = not op.lower
+
+    def cholesky(a):
+        return torch.linalg.cholesky(a, upper=upper)
+
+    return cholesky
+
+
+@pytorch_funcify.register(Solve)
+def pytorch_funcify_Solve(op, **kwargs):
+    lower = False
+    if op.assume_a != "gen" and op.lower:
+        lower = True
+
+    def solve(a, b):
+        if lower:
+            return torch.linalg.solve(torch.tril(a), b)
```

Comment:
I don't think we need to do this. At worst it will introduce silent bugs in user code (because `solve` doesn't do any type checking w.r.t the keywords and args), at best it introduces an extra `tril` `Op` for nothing.

Reply:
`torch.solve` doesn't have any keywords, so I suggest just directly returning the base case.

Reply:
`torch.solve` is deprecated.

Reply:
I mean `torch.linalg.solve`, which also has none of the scipy solver keywords (most importantly `assume_a`)